TFT/src/User/API/Touch_Screen.c

```diff
@@ -211,12 +213,13 @@ void TS_Calibrate(void)
         GUI_DrawPoint(LCD_X[tp_num], LCD_Y[tp_num] - i);
       }
 
-      while (TS_IsPressed() == false);
-
+      while (!TS_IsPressed());
+      Buzzer_AddSound(BUZZER_FREQUENCY_HZ, BUZZER_FREQUENCY_DURATION_MS);
```

Comment:
empty line before this buzzer call. `while`, `if`, `switch` etc. must be always separated by empty line